e2e/test/iothub/service/RegistryE2ETests.cs

```diff
@@ -574,6 +574,24 @@ public async Task RegistryManager_DeviceTwinLifecycle()
             }
         }
 
+        [LoggedTestMethod]
+        public async Task DevicesClient_GetStatistics()
+        {
+            using var serviceClient = new IotHubServiceClient(TestConfiguration.IoTHub.ConnectionString);
+
+            // No great way to test the accuracy of these statistics, but making the request successfully should
+            // be enough to indicate that this API works as intended
+            ServiceStatistics serviceStatistics = await serviceClient.Devices.GetServiceStatisticsAsync().ConfigureAwait(false);
```

Comment:
Unrelated to the changes in this PR, but what is `Get ServiceStatistics` supposed to return? Looking at the value below (.ConnectedDeviecCount), it looks like it returns device statistics that are stored at service end? Is that accurate? If yes, then should we call it "Devices.ServiceStatistics" or "Devices.Statistics"? The term "ServiceStatistics" made me think that I should expect some "service" related statistics; i.e. hub units, scale, etc.

Reply:
Good point. This may be worth renaming. I'll follow up on this in a later PR